.github/workflows/release-staging.yml

```diff
@@ -3,6 +3,7 @@ name: Deploy Components to Staging
 on:
   workflow_dispatch: # This allows manual triggering
   push:
+  pull_request:
```

Comment:
I think we should remove the automated, rather make it only manual, because let's say I shared a URL for testing, and someone else create a PR, then my changes will be removed, I will have to again deploy it, let's only keep manual, so whenever we have to share the link, we manually go and deploy